site/content/docs/cli/project-structure.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Project Structure
+weight: 3
+menu:
+  docs:
+    parent: "CLI"
+---
+
+This is how a typical project using the Checkly CLI is organized. Most files, directories and paths are conventions you
```

Comment:
This is an undefined `this` in writing. This page? This example? maybe we can rephrase.